x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/plotUtils.js

```diff
@@ -80,7 +80,7 @@ export function getPlotValues(
     XY_MARGIN,
     XY_HEIGHT: height || XY_HEIGHT,
     XY_WIDTH: width,
-    ...(stacked ? { stackBy: 'y' } : {})
+    stackBy
```

Comment:
unfortunately as soon as this value is set, weird things happen (with both 'x' and 'y' as a value).